components/crud-web-apps/volumes/frontend/src/app/pages/form/form-default/form-default.component.html

```diff
@@ -31,16 +34,14 @@
       [sizeControl]="formCtrl.get('size')"
       min="1"
       step="1"
-      label="{{ 'volume.form.volumeSizeGi' | translate }}"
+      label="Volume size in Gi"
```

Comment:
This label needs to be marked for translation.

Reply:
Pushed a commit